packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md

```diff
@@ -3,5 +3,62 @@ id: Toggle group
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/community#get-in-touch)
+A **toggle group** is a group of controls that can be used to quickly switch between actions or states. 
+
+## Elements
+Toggle items can consist of icons, text, or both. Styling of toggles is dependent on the background color. 
+
+<img src="./img/elements-items.png" alt="Toggle group elements" width="1536"/>
+
+## Toggle group types
+
+1. Single select toggle group: Use to toggle between multiple items
```

Comment:
Would add a link to each type, that would scroll the user down to where that content is on the page!

Reply:
fixed!